data/Acute_Toxicity_LD50/meta.yaml

```diff
@@ -0,0 +1,45 @@
+name: ld50_zhu
+description: Acute toxicity LD50 measures the most conservative dose that can lead
+  to lethal adverse effects. The higher the dose, the more lethal of a drug.
+targets:
+- id: acute_toxicity
+  description: Acute Toxicity LD50.
+  units: ld50
+  type: continuous
+  names:
+  - Acute Toxicity LD50
+  - ld50
+  - conservative dose that can lead to lethal adverse effects.
+  - Rat Acute Toxicity by Oral Exposure
+  - Toxicity
```

Comment:
@MicPie I'm really thinking more and more that our model might benefit from also having descriptions about what those things mean. We should decide of how we approach this and what we aim for as "names" in the prompts. I'm hesitant with general things such as "toxcitiy" as they can mean too many things and we now have the chance to really create a high-quality dataset